### PR TITLE
Update memory settings for Docker for Mac

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

**Comment by electrum** [comment]
Should we specifically say 4GB is recommended?

**Comment by sanjay990** [comment]
@electrum : Yes, I think that's fine. I'll update

**Comment by sanjay990** [comment]
@electrum : Updated

File: presto-product-tests/README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
 They should point to your external IP address (shown by `ifconfig` on your Mac (not inside docker)).
 
+* The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap.
+We recommend at least 4GB for such profiles. You can increase docker memory by going to Docker
```
Comment (findepi) [inline presto-product-tests/README.md]: When i tried to run single node tests with `run_on_docker` i needed to slide up to 7 GB for them to pass.
 They definitely weren't passing at 4 GB.. but this was 1-2 mo ago, so maybe something changed or it was just mine machine.

File: presto-product-tests/README.md [hunk 2]
```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
```
Comment (findepi) [inline presto-product-tests/README.md]: That's kind of weird. First, not all these are needed. And any needed looks like a test scaffolding deficiency... but this is of topic I know

File: presto-product-tests/README.md (hunk 1 above)
Comment (sanjay990) [inline presto-product-tests/README.md]: @electrum tried with 4 GB and it worked fine. We are also planning to try on @maciejgrzybek's machine.

Comment (findepi) [inline presto-product-tests/README.md]: @sanjay990 what does it mean "try"? Checking tests start up or letting them all pass?
Which configuration are you checking?

(`./presto-product-tests/bin/run_on_docker.sh singlenode -g tpcds -x quarantine -t q98` didn't pass locally with 4 GB and no other containers prior running (presto got killed) - and this is only single test.)

Comment (findepi) [inline presto-product-tests/README.md]: ... `./presto-product-tests/bin/run_on_docker.sh singlenode -g tpcds -x quarantine` not passing at 6 GB (presto killed), seems working at 7 GB

Comment (sanjay990) [inline presto-product-tests/README.md]: @findepi : This fix is related to https://github.com/prestodb/presto/issues/7676
We checked for singlenode-ldap and made sure tests pass. But I see your point.
Even with singlenode profile you might need more memory depending on tests you want to run.
We recommend 4 GB as it probably works in most cases and user should increase further if needed.

**Comment by findepi** [comment]
Thanks for explanation. The wording could be more explicit that 4 GB is
not going to suffice in many cases and it is no error.
E.g. "you may need 4-8 GB or even more to run certain tests"

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**